src/Epicea/EpMonitor.class.st

```diff
@@ -383,7 +386,7 @@ EpMonitor >> handleAnyErrorDuring: aBlock [
 EpMonitor >> handleError: error [
 
 	self debugMode 
-		ifTrue: [ error signal ]
+		ifTrue: [ error pass ]
```

Comment:
This is a small unrelated fix (low impact... user only see it if you enable debugging with `EpMonitor current debugMode: true`